content/projects/simple-website/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+content_type: project
+flavours:
+- none
+learning_outcomes:
+- web_dev_hosting
+- web_dev_responsive_page_designs
+prerequisites:
+  hard:
+  - topics/what-to-put-in-your-cv
+ready: true
+story_points: 3
+submission_type: link
+tags:
+- html
+- css
+- responsive web design
+title: Simple Website - Build your first personal website
+---
+For the learnership you will create a website with your CV and a portfolio of your work. Create the website and host it on Github. You will be expected to keep your portfolio up to date throughout the learnership.
+
+## Instructions
+
+Create a personal website to showcase your talents and ambitions.
+
+The content and information you choose to display is up to you. There should be some information about your skills, interests, and previous experience, as well as information on how to get in touch with you. Think about the user experience when designing your layout and navigation.
+
+The HTML and CSS should be developed in a logical way with good naming conventions that would allow another developer to easily understand and build upon your code. There is something called "Semantic Markup" - read all about it [here](https://seekbrevity.com/semantic-markup-important-web-design/).
+
+You can also make use of this cool [markup validation tool](https://validator.w3.org/) to validate the HTML code used in your website.
```

Comment:
Should data scientists care about this? We just need them to do enough to pass their ncit strandard.